src/assets/styles/components/_checkbox.css

```diff
@@ -12,6 +12,7 @@
         calc(0.125rem * var(--fl-enhance-font-size-factor, 1))
         var(--fl-fgColor, var(--color-indigo-700));
     inline-size: calc(1.25rem * var(--fl-enhance-font-size-factor, 1));
+    min-width: calc(1.25rem * var(--fl-enhance-font-size-factor, 1));
```

Comment:
Remove flex from parent, use `vertical-align: middle` on checkboxes and labels, and use a combination of `text-indent` and `margin-block-start` on the parent to give the labels a hanging indent when they take up more than one line.

Reply:
Hey Ned, I took a slight different approach here to achieve what we talked about. Please let me know if you think my approach isn't working.